webrtc.html

```diff
@@ -4624,13 +4624,14 @@ <h2>Methods</h2>
         </div>
         <p>For the purposes of this API, the <a>[[\Certificate]]</a> slot
         contains unstructured binary data.</p>
-        <p>Note that an <code>RTCCertificate</code> might not directly hold
-        private keying material, this might be stored in a secure module.</p>
         <p>The <code>RTCCertificate</code> object can be stored and retrieved
-        from persistent storage by an application. When a <a>user agent</a> is
-        required to obtain a structured clone [[!HTML51]] of an
-        <code>RTCCertificate</code> object, it performs the following
-        steps:</p>
+        from persistent storage by an application. Since the <code>RTCCertificate</code>
+        represents a handle to both the certificate and the private key,
+        implementations MUST support storing and retrieving both the certificate
+        and private key, even if the <code>RTCCertificate</code> stores the private
+        key in a secure module. When a <a>user agent</a> is required to obtain a
```

Comment:
Suggested text:
Note that there is no way to access the private key, so it doesn't matter to the user whether the private key is stored directly or if it is stored as a reference to some other key storage such as a KSM.